UPGRADING

```diff
@@ -160,8 +160,8 @@ PHP 8.3 UPGRADE NOTES
     RFC: https://wiki.php.net/rfc/deprecations_php_8_3#mt_rand_php
 
 - Standard:
-  . The assert_option() function is now deprecated.
-  . The ASSERT_ACTIVE, ASSERT_BAIL,ASSERT_CALLBACK, ASSERT_EXCEPTION, and
+  . The assert_options() function is now deprecated.
+  . The ASSERT_ACTIVE, ASSERT_BAIL, ASSERT_CALLBACK, ASSERT_EXCEPTION and
```

Comment:
isn't the oxford comma needed before the last and?

Reply:
Good question. That's a choice though, it's not necessarily grammatically correct (or better). Is this being used consistently in the repo ?

Reply:
Brought the comma back.